<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="mainStackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="loginPage">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="loginAsClientButton">
          <property name="text">
           <string>Login as Client</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loginAsStaffButton">
          <property name="text">
           <string>Login as Staff</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mainPage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="mainTabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="animalTab">
           <attribute name="title">
            <string>Animals</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QStackedWidget" name="animalsStackedWidget">
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="animalViewPage">
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <widget class="QPushButton" name="addAnimalButton">
                    <property name="text">
                     <string>Add Animal</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="editAnimalButton">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Edit Animal</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QListView" name="animalListView">
                  <property name="editTriggers">
                   <set>QAbstractItemView::NoEditTriggers</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="animalFormPage">
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <layout class="QFormLayout" name="animalForm">
                  <property name="fieldGrowthPolicy">
                   <enum>QFormLayout::ExpandingFieldsGrow</enum>
                  </property>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Animal Type</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Breed</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Color</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="0">
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>Age</string>
                    </property>
                   </widget>
                  </item>
                  <item row="5" column="0">
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>Gender</string>
                    </property>
                   </widget>
                  </item>
                  <item row="6" column="0">
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>Weight</string>
                    </property>
                   </widget>
                  </item>
                  <item row="7" column="0">
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>Height</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QComboBox" name="animalFormTypeInput">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="animalFormBreedInput">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QLineEdit" name="animalFormColorInput"/>
                  </item>
                  <item row="4" column="1">
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                     <widget class="QDoubleSpinBox" name="animalFormAgeInput">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="singleStep">
                       <double>0.500000000000000</double>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_8">
                      <property name="maximumSize">
                       <size>
                        <width>45</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>years</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="5" column="1">
                   <widget class="QComboBox" name="animalFormGenderInput">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <item>
                     <property name="text">
                      <string>Male</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Female</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item row="6" column="1">
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QDoubleSpinBox" name="animalFormWeightInput">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="maximum">
                       <double>500.000000000000000</double>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_10">
                      <property name="maximumSize">
                       <size>
                        <width>20</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>kg</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="7" column="1">
                   <layout class="QHBoxLayout" name="horizontalLayout_7">
                    <item>
                     <widget class="QDoubleSpinBox" name="animalFormHeightInput">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="maximum">
                       <double>500.000000000000000</double>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_11">
                      <property name="maximumSize">
                       <size>
                        <width>20</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>cm</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>Name</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="animalFormNameInput"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QPushButton" name="animalFormCancelButton">
                    <property name="text">
                     <string>Cancel</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="animalFormSaveButton">
                    <property name="text">
                     <string>Save</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
